style.css

```diff
@@ -63,3 +61,194 @@ li {
   line-height: 28px;
   margin: 1.5em 25px;
 }
+
+.recent-works {
+  display: grid;
+  width: 90%;
+  grid-template-columns: 1fr;
+  grid-template-rows: 200px repeat(5, 6fr);
+  margin: 0 auto;
+  text-align: center;
+}
+
+h2 {
+  font-size: 40px;
+  color: #172b4d;
+}
+
+.recent-works h2:first-child::after {
+  content: '';
+  display: block;
+  position: relative;
+  height: 4px;
+  width: 48px;
+  margin: 0 auto;
+  top: 20px;
+  z-index: -1;
+  border-radius: 24px;
+  background: #36b37e;
+}
+
+.myworks-card {
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+  padding: 0;
+  margin: 1em auto;
+  width: 327px;
+  height: 474px;
+  border: 1px lightgray solid;
+  border-radius: 5px;
+}
+
+.myworks-thumbnail {
+  grid-column: 1/-1;
+  grid-row: 2/3;
+  overflow: hidden;
+  box-sizing: border-box;
+}
+
+.myworks-thumbnail img {
+  width: 100%;
+  min-height: 220px;
+  object-fit: cover;
+  background-color: #d0d9d4;
+  border-radius: 2px;
+}
+
+.work-title {
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 1.5em;
+  color: #3a4a42;
+  text-align: center;
+  grid-column: 1/3;
+  grid-row: 3/4;
+}
+
+.work-languages li {
+  background: #ebf0ee;
+  padding: 0.6em;
+  margin: 0 0.5em;
+  border-radius: 4px;
+  font-size: 12px;
+  font-weight: 600;
+}
+
+.attribute-list li {
+  background-color: #fff;
+  color: #36b37e;
+  font-size: 14px;
+  font-weight: 600;
+  padding: 1em 0.5em;
+  margin: 1em 0.5em;
+  border-radius: 5px;
+}
+
+.myworks-card > .work-languages > li {
+  background: #ebf0ee;
+  font-family: Inter, sans-serif;
+  font-size: 16px;
+  font-weight: 600;
+  padding: 0.5em;
+  margin: 0 0.5em;
+  border-radius: 4px;
+}
+
+.work-languages {
+  display: flex;
+  justify-content: space-around;
+  list-style: none;
+  padding: 0.5em;
+}
+
+button {
+  margin-top: 1.5em;
+  margin-bottom: 1.5em;
+  border: none;
+  border-radius: 4px;
+  width: fit-content;
+  align-self: center;
+  background: #36b37f;
+  color: white;
+  font-weight: 500;
+  font-size: 17px;
+  letter-spacing: 2px;
+  padding: 12px;
+}
+
+button:hover {
+  background: #36b37f;
+  box-shadow: 1px 8px 16px rgba(54, 179, 127, 0.24);
+  border-radius: 4px;
+}
+
+button:active {
+  background: #008552;
+}
+
+.about-me {
```

Comment:
- [ ] the current design of your about me section doesn't match the figma template design see images below for reference and this is so because of the fixed width css styling you use kindly please use `%` which are responsive to screen sizes rather than `px` which are fixed.
> - your design
![s-6-4](https://user-images.githubusercontent.com/89055502/235276524-a9c09043-f04f-48fa-9214-9a92cedd45fc.png)
> - figma design

![f-6-4](https://user-images.githubusercontent.com/89055502/235276669-8539bc7e-9bfd-45d2-a10b-14eae71a3753.png)